gui/executionGraphGui/client/apps/eg/src/app/model/Node.ts

```diff
@@ -26,36 +26,19 @@ export class UIProps {
 /**
  * Unique Identifier for a `Node`.
  *
- * @export
- * @class NodeId
- * @extends {Long}
  */
 export class NodeId extends Long {
```

Comment:
This is differerent to the ConnectionId. Why not just defining a type alias too? 2 Solutions for the same concept here...